client/packages/theia-integration/README.md

```diff
@@ -0,0 +1,3 @@
+# GLSP - Theia Integration
+
+This package contais the glue code necessary to enable the integration of the  GLSP client(s) in Eclipse Theia. (see also https://github.com/eclipsesource/GraphicalServerProtocol)
```

Comment:
There is a space too much between the and GLSP client.
Also I'd write "integration of GLSP clients" instead of the "...(s)"
Finally I'd put the period after the closing bracket.